lib/pos_snapshot_entries.go

```diff
@@ -675,6 +675,268 @@ func DBPutSnapshotValidatorSetTotalStakeAmountNanosWithTxn(
 	return DBSetWithTxn(txn, snap, key, VariableEncodeUint256(globalActiveStakeAmountNanos))
 }
 
+//
+// SnapshotStakeToReward
+//
+
+type SnapshotStakeMapKey struct {
+	SnapshotAtEpochNumber uint64
+	ValidatorPKID         PKID
+	StakerPKID            PKID
+}
+
+// This is a bare bones in-memory only construct used to capture the ValidatorPKID,
+// StakerPKID, and StakeAmountNanos from a StakeEntry that has been snapshot. We
```

Comment:
```suggestion
// StakerPKID, and StakeAmountNanos from a StakeEntry that has been snapshotted. We
```

Reply:
Will fix in next PR so we can avoid one extra CI run.